doc/source/index.rst

```diff
@@ -3,268 +3,74 @@ Dask Cloud Provider
-
-Here is the full minimal IAM policy that you need to create the whole cluster:
+    cluster.close()
 
-.. code-block:: json
-   
-   {
-       "Statement": [
-           {
-               "Action": [
-                   "ec2:AuthorizeSecurityGroupIngress",
-                   "ec2:CreateSecurityGroup",
-                   "ec2:CreateTags",
-                   "ec2:DescribeNetworkInterfaces",
-                   "ec2:DescribeSubnets",
-                   "ec2:DescribeVpcs",
-                   "ec2:DeleteSecurityGroup",
-                   "ecs:CreateCluster",
-                   "ecs:DescribeTasks",
-                   "ecs:ListAccountSettings",
-                   "ecs:RegisterTaskDefinition",
-                   "ecs:RunTask",
-                   "ecs:StopTask",
-                   "ecs:ListClusters",
-                   "ecs:DescribeClusters",
-                   "ecs:DeleteCluster",
-                   "ecs:ListTaskDefinitions",
-                   "ecs:DescribeTaskDefinition",
-                   "ecs:DeregisterTaskDefinition",
-                   "iam:AttachRolePolicy",
-                   "iam:CreateRole",
-                   "iam:TagRole",
-                   "iam:PassRole",
-                   "iam:DeleteRole",
-                   "iam:ListRoleTags",
-                   "iam:ListAttachedRolePolicies",
-                   "iam:DetachRolePolicy",
-                   "logs:DescribeLogGroups"
-               ],
-               "Effect": "Allow",
-               "Resource": [
-                   "*"
-               ]
-           }
-       ],
-       "Version": "2012-10-17"
-   } 
-   
-If you specify all of the resources yourself you will need a minimal policy of:
+.. warning::
 
-.. code-block:: json
-   
-   {
-       "Statement": [
-           {
-               "Action": [
-                   "ec2:CreateTags",
-                   "ec2:DescribeNetworkInterfaces",
-                   "ec2:DescribeSubnets",
-                   "ec2:DescribeVpcs",
-                   "ecs:DescribeTasks",
-                   "ecs:ListAccountSettings",
-                   "ecs:RegisterTaskDefinition",
-                   "ecs:RunTask",
-                   "ecs:StopTask",
-                   "ecs:ListClusters",
-                   "ecs:DescribeClusters",
-                   "ecs:ListTaskDefinitions",
-                   "ecs:DescribeTaskDefinition",
-                   "ecs:DeregisterTaskDefinition",
-                   "iam:ListRoleTags",
-                   "logs:DescribeLogGroups"
-               ],
-               "Effect": "Allow",
-               "Resource": [
-                   "*"
-               ]
-           }
-       ],
-       "Version": "2012-10-17"
-   } 
+   Cluster managers will attempt to automatically remove hanging cloud resources on garbage collection if the cluster
+   object is destroyed without calling ``cluster.close()``, however this is not guaranteed.
 
-GPU Support
```

Comment:
Maybe we should keep some information around about launching GPU instances and link to the specific examples like ecscluster+rapids ?

Reply:
This ECS example was moved into the docstring of `ECSCluster`. We could add a general GPU section to the index though.